notebooks/Welcome to MONAI Cloud APIs.ipynb

```diff
@@ -29,17 +29,39 @@
     "\n",
     "- [Dataset Preparation and Model Selection](./Dataset%20Creation%20and%20Model%20Selection.ipynb)\n",
     "\n",
-    "### 3. Annotation and Continuous Learning Jobs\n",
+    "\n",
+    "### 3. Training a Segmentation Bundle\n",
```

Comment:
Do we want a different name than "Training a Segmentation Bundle"? Maybe something like "Training with a Provided Network".

Or potentially a different name.  Part of this is to differentiate between networks we provide vs custom bundles.

Reply:
Sure we can rename it